src/Microsoft.Health.Dicom.Core.UnitTests/Features/Resources/Retrieve/LazyTransformReadOnlyStreamTests.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Health.Dicom.Core.UnitTests.Features.Resources.Retrieve
 {
     public class LazyTransformReadOnlyStreamTests
     {
-        private static readonly RecyclableMemoryStreamManager _recyclableMemoryStreamManager = new RecyclableMemoryStreamManager();
+        private static readonly RecyclableMemoryStreamManager RecyclableMemoryStreamManager = new RecyclableMemoryStreamManager();
```

Comment:
Do we want `static readonly` fields to use the same rules:
- `const` (PascalCase)
- `private` (`"_example"`)
- `static private` (`"s_example"`)

Reply:
Combining `const` with `static` `readonly` styles